<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>AdminWindow</class>
    <widget class="QMainWindow" name="AdminWindow">
        <property name="geometry">
            <rect>
                <x>0</x><y>0</y><width>1000</width><height>600</height>
            </rect>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout">
                <item>
                    <widget class="QTabWidget" name="tabWidget">
                        <!-- 书籍管理 Tab -->
                        <widget class="QWidget" name="tabBooks">
                            <attribute name="title"><string>书籍管理</string></attribute>
                            <layout class="QVBoxLayout">
                                <item>
                                    <widget class="QTableView" name="tableViewBooks"/>
                                </item>
                                <item>
                                    <layout class="QHBoxLayout">
                                        <item>
                                            <widget class="QPushButton" name="btnAddBook">
                                                <property name="text"><string>新增</string></property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QPushButton" name="btnEditBook">
                                                <property name="text"><string>编辑</string></property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QPushButton" name="btnDeleteBook">
                                                <property name="text"><string>删除</string></property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QPushButton" name="btnSearchBook">
                                                <property name="text"><string>搜索</string></property>
                                            </widget>
                                        </item>
                                    </layout>
                                </item>
                            </layout>
                        </widget>
                        <!-- 用户管理 Tab -->
                        <widget class="QWidget" name="tabUsers">
                            <attribute name="title"><string>用户管理</string></attribute>
                            <layout class="QVBoxLayout">
                                <item>
                                    <widget class="QTableView" name="tableViewUsers"/>
                                </item>
                                <item>
                                    <layout class="QHBoxLayout">
                                        <item>
                                            <widget class="QPushButton" name="btnAddUser">
                                                <property name="text"><string>新增</string></property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QPushButton" name="btnEditUser">
                                                <property name="text"><string>编辑</string></property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QPushButton" name="btnDeleteUser">
                                                <property name="text"><string>删除</string></property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QPushButton" name="btnSearchUser">
                                                <property name="text"><string>搜索</string></property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QPushButton" name="btnResetPassword">
                                                <property name="text"><string>重置密码</string></property>
                                            </widget>
                                        </item>
                                    </layout>
                                </item>
                            </layout>
                        </widget>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar"/>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
